data/agent-rinkeby.conf

```diff
@@ -0,0 +1,25 @@
+log-level = "debug"
+deployment-dir = "/data/deployments/rinkeby"
+fill-wait-time = 120
+
+[account]
+path = "/data/account/keyfile.json"
+password = "123"
```

Comment:
why do we have "123" here and in the mainnet file it is empty? Wouldbe nice to stays consistent

Reply:
> why do we have "123" here and in the mainnet file it is empty? Wouldbe nice to stays consistent

Nice catch, thanks! Removed.